.travis.yml

```diff
@@ -2,6 +2,8 @@ language: rust
 env:
   global:
     - secure: g4kCg8twONwKPquuJmYrvGjo2n0lNtWTbyzFOITNn8FgCxNK2j38Qc9/UhErTR3g3rDjVzsTHZ8FTH7TJZrOK1Nzz90tJG6JHqUv77ufkcBlxgwwjilOz84uQhkDTMpLitMEeQDLEynKeWbxrjtc5LIpjEkxOPk5eiqwzKRN14c=
+before_script:
+  - mkdir -p .cargo && echo 'paths = ["python27-sys"]' > .cargo/config
```

Comment:
Interesting! I didn't know about cargo path overrides.

Is there any reason not to use these when working with the rust-cpython repository? I think we might want to commit `.cargo/config` to version control.